### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by SaboorSohaib**
### The following requirements are implemented. 
- Add box-shadow text-align to the button.
- Add property to the active status.

Kindly check my project if there is any error.
Thanks!

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi @SaboorSohaib :wave:  

Good job so far!
Just one thing to fix.

## Highlights
 - [X] You have added the hover effect on almost all of the buttons.

## Required Changes ♻️

Kindly ensure the hover effect works and add the property ***cursor: pointer*** on the last button - Get in touch.
Although the hover effect is implemented in your CSS, it is not working.

That's it. :slightly_smiling_face: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -719,6 +728,7 @@ ul li {
   text-align: center;
   letter-spacing: 0.03em;
   color: rgb(255, 255, 255);
+  cursor: pointer;
 }
 
 .contact-btn:hover {
```
Comment (ElsonOtake) [inline style.css:846]: - [ ] I think it's better to use the properties for hover provided by the Figma template. It's a project requirement. There are changes in box-shadow and text-align.
- [ ] Same thing for "Get my resume" button.
- [ ] Same thing for "See project" button.

File: style.css [hunk 2]
```diff
@@ -733,15 +743,15 @@ ul li {
   border-radius: 4px;
 }
 
-.contact-btn:enabled {
+.contact-btn:active {
```
Comment (ElsonOtake) [inline style.css:857]: - [ ] I think it's better to use the properties for active provided by the Figma template. It's a project requirement.
- [ ] Same thing for "Get my resume" button.
- [ ] Same thing for "See project" button.

Screen Shot 2022-08-26 at 10 38 41 PM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @SaboorSohaib, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

woohoo-youre-almost

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] I think it's better to write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. More information in [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md).

## Optional suggestions

- [OPTIONAL] Please use the background provided by the Figma template to the "Get in touch" section. I consider that optional because was not requested by the previous reviewer.
- [OPTIONAL] Please use the background provided by the Figma template to the "About me" section. I consider that optional because was not requested by the previous reviewer.  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaboorSohaib** [comment]
Hi!
I did not understand the required changes. Is it enough to update the pull
request title or there is something else to do?
I think the below comments are optional.

**Review by Ismailco — APPROVED** [review]
### Hi @SaboorSohaib, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**